docs/content/reboot.md

```diff
@@ -301,3 +301,13 @@ HTML5 adds [a new global attribute named `[hidden]`](https://developer.mozilla.o
 {% endcallout %}
 
 To merely toggle the visibility of an element, meaning its `display` is not modified and the element can still affect the flow of the document, use [the `.invisible` class]({{ site.baseurl }}/components/utilities/#invisible-content) instead.
+
+## Touch responsiveness optimization
+
+Traditionally, browsers on touchscreen devices have a delay of approximately 300ms between the end of a "tap" – the moment when a finger/stylus is lifted from screen – and the `click` event being fired. This delay is necessary for these browsers to correctly handle "double-tap to scroll" gestures without prematurely triggering actions or links after the first "tap", but it can make your site feel slightly sluggish and unresponsive.
+
+Most mobile browsers automatically optimize away this 300ms delay for sites that use the `width=device-width` property as part of their [responsive meta tag]({{ site.baseurl }}/getting-started/introduction/#responsive-meta-tag) (as well as for sites that disable zooming, for instance with `user-scalable=no`, though this practice is strongly discouraged for accessibility and usability reasons). The only exception here is currently iOS Safari (and any other WebView-based browser), which requires additional scripts such as [FastClick](https://github.com/ftlabs/fastclick) to explicitly work around the delay (though this is likely to change in future versions of iOS - see [Bug 150604 - Implement viewport-width-based fast-click heuristic](https://bugs.webkit.org/show_bug.cgi?id=150604)).
+
+On touch-enabled laptop/desktop devices, the majority of browsers don't have any "double-tap to scroll" functionality, and therefore don't have any delay. The exceptions here are IE11 and Microsoft Edge, which do allow the user to double-tap. As the [responsive meta tag]({{ site.baseurl }}/getting-started/introduction/#responsive-meta-tag) is ignored by desktop browsers (including IE11 and Microsoft Edge), using `width=device-width` will have no effect on the 300ms delay. For this reason, Bootstrap explicitly uses the `touch-action:manipulation` CSS property (defined as part of the [Pointer Events](http://www.w3.org/TR/pointerevents/#the-touch-action-css-property) specification) on all actionable elements (such as buttons and links). This property essentially disables double-tap functionality on those elements, eliminating the 300ms delay.
```

Comment:
> The exceptions here are IE11 and Microsoft Edge, which do allow the user to double-tap.

I would either add "to scroll" or rephrase to something like "which do implement this feature".